pkg/test/fuzz/generate.go

```diff
@@ -46,6 +46,20 @@ func fillWithRandom0(t *testing.T, randStream *rand.Rand, msg protoreflect.Messa
 		return
 	}
 
+	if string(descriptor.FullName()) == "google.protobuf.Timestamp" {
+		count := randStream.Intn(10)
+		// Bias to zero
+		if count > 4 {
+			return
+		}
+		// Generate a "reasonable" timestamp
```

Comment:
"Unreasonable" dates are (literally) billions of years in the future.    Not a problem in practice, but the fuzzer finds these problems pretty fast if we don't stop them!